internal/provider/resource_vault_cluster_admin_token.go

```diff
@@ -33,6 +40,11 @@ func resourceVaultClusterAdminToken() *schema.Resource {
 				ValidateDiagFunc: validateSlugID,
 			},
 			// computed outputs
+			"created_at": {
+				Description: "The time that the admin token was created.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
```

Comment:
You might already have this on a local branch, but I think we still want the `token` below to be `Sensitive`

Reply:
Ooh yes, I double-checked this for the data source PR but missed it on this one - good catch!